src/main/java/org/broadinstitute/hellbender/engine/VariantWalkerBase.java

```diff
@@ -95,6 +94,15 @@ public final SAMSequenceDictionary getBestAvailableSequenceDictionary() {
     protected abstract Spliterator<VariantContext> getSpliteratorForDrivingVariants();
 
     /**
+     * When performing a query on the primary variant input how many overlapping records should be cached when querying.
+     * Subclasses can set this value by overriding this method.
+     * @return the number of bases ahead of a query to prefetch
+     */
+    protected int getVariantCacheLookAheadBases(){
```

Comment:
Name of this method should make it clear that it applies only to the driving input (eg., `getDrivingVariantsCacheLookaheadBases()`)